inappmessaging/USERGUIDE.md

```diff
@@ -97,12 +100,12 @@ Please use the Create Campaign page to create, edit, save as draft, launch, canc
   * Define campaign message **Start Date** and **End Date**.
   * **Choose App** and **Versions** - Choose which app in your app group you are going to apply to the campaign.
   * Click "+" to add multiple apps.
-  * **Target Users** - You can target all your audience with select All Users, you can also target users by upload a CSV file with a list of Easy Ids or User Ids.
+  * **Target Users** - You can target all your audience with select All Users, you can also target users by uploading a CSV file with a list of Easy IDs or User IDs.
```

Comment:
I'm not sure we should have dashboard user guide in the github readme because the dashboard is internal. We don't have dashboard details in the iOS readme.

Reply:
My understanding is that dashboard is not internal. It can be accessed without INTRA.

Reply:
removed